controllers/main.go

```diff
@@ -1692,46 +1729,70 @@ func (controller *MainController) Tickets(c web.C, r *http.Request) (string, int
 		return controller.Parse(t, "main", c.Env), http.StatusInternalServerError
 	}
 
+	log.Debugf(":: StakePoolUserInfo (msa = %v) execution time: %v",
+		user.MultiSigAddress, time.Since(start))
+
+	_, height, err := w.GetBestBlock()
+	if err != nil {
+		log.Infof("RPC GetBestBlock failed: %v", err)
+		session.AddFlash("Unable to get best block height", "main")
+		c.Env["Flash"] = session.Flashes("main")
+		return controller.Parse(t, "main", c.Env), http.StatusInternalServerError
+	}
+	minVotedHeight := height - controller.maxVotedAge
+
 	// If the user has tickets, get their info
 	if spui != nil && len(spui.Tickets) > 0 {
-		for idx, ticket := range spui.Tickets {
+		for _, ticket := range spui.Tickets {
 			switch ticket.Status {
 			case "live":
-				ticketInfoLive[idx] = TicketInfoLive{
-					Ticket:       ticket.Ticket,
+				ticketInfoLive = append(ticketInfoLive, TicketInfoLive{
 					TicketHeight: ticket.TicketHeight,
-				}
+					Ticket:       ticket.Ticket,
+				})
 			case "expired":
-				ticketInfoExpired[idx] = TicketInfoHistoric{
+				ticketInfoExpired = append(ticketInfoExpired, TicketInfoHistoric{
 					Ticket:        ticket.Ticket,
 					SpentByHeight: ticket.SpentByHeight,
 					TicketHeight:  ticket.TicketHeight,
-				}
+				})
 			case "missed":
-				ticketInfoMissed[idx] = TicketInfoHistoric{
+				ticketInfoMissed = append(ticketInfoMissed, TicketInfoHistoric{
 					Ticket:        ticket.Ticket,
 					SpentByHeight: ticket.SpentByHeight,
 					TicketHeight:  ticket.TicketHeight,
-				}
+				})
 			case "voted":
-				ticketInfoVoted[idx] = TicketInfoHistoric{
-					Ticket:        ticket.Ticket,
-					SpentBy:       ticket.SpentBy,
-					SpentByHeight: ticket.SpentByHeight,
-					TicketHeight:  ticket.TicketHeight,
+				numVoted++
+				if int64(ticket.SpentByHeight) >= minVotedHeight {
+					ticketInfoVoted = append(ticketInfoVoted, TicketInfoHistoric{
+						Ticket:        ticket.Ticket,
+						SpentBy:       ticket.SpentBy,
+						SpentByHeight: ticket.SpentByHeight,
+						TicketHeight:  ticket.TicketHeight,
+					})
 				}
 			}
 		}
 
-		for idx, ticket := range spui.InvalidTickets {
-			ticketInfoInvalid[idx] = TicketInfoInvalid{ticket}
+		for _, ticket := range spui.InvalidTickets {
+			ticketInfoInvalid = append(ticketInfoInvalid, TicketInfoInvalid{ticket})
 		}
 	}
 
+	// Sort live tickets
+	sort.Sort(ByTicketHeight(ticketInfoLive))
```

Comment:
Might as well comment this.  At best it will help accelerate the JS sorting by providing a pre-sorted list.